src/pages/dashboard/component.js

```diff
@@ -42,17 +42,26 @@ export const Dashboard = ({fetchChallenges, challenges, user}) => {
       <Tabs defaultActiveKey={tab} onChange={setTab} centered={true}>
         <TabPane tab="Active Challenges" key="active">
           <section className={styles.challenges}>
-            {challenges &&
-              challenges.map((challenge, i) => (
-                <Challenge
-                  key={i}
-                  data={{...challenge, username: user.actorHandle.assetId.id}}
-                  challengeIndex={i}
-                />
-              ))}
+            {challenges.active.map((challenge, i) => (
+              <Challenge
+                key={i}
```

Comment:
please, use id of the challenge
Example is here https://github.com/StepanNaryshkovEleks/pwa/blob/main/src/pages/challenges/component.js#L76